websockets/websockets.go

```diff
@@ -51,8 +51,8 @@ type ReadyState uint8
 const (
 	// CONNECTING is the state while the web socket is connecting
 	CONNECTING ReadyState = iota
-	// OPEN is the state after the websocket is established and before it starts closing
-	OPEN
+	// OPENED is the state after the websocket is established and before it starts closing
+	OPENED
```

Comment:
This is named after the name in the specification and I would prefer to keep it that way

Reply:
Yeah, I know, but specifications force us to have this inside the JS whenever this is pure go internals. So keeping in mind the frequency of internal usage, it makes sense to reserve for the go constant `OPEN` an event, not the state. 

If that argument is not enough, I can revert it and make the event's constant prefixable, like `EVENT_OPEN`, `EVENT_CLOSE`, and so on.

WDYT?

Reply:
Went with another suggestion (package `event`)